desktop/apps/consign/test/reducers.js

```diff
@@ -144,7 +294,8 @@ describe('Reducers', () => {
           ]
           store.dispatch(actions.logIn({ email: 'sarah@sarah.com', password: '1234' })).then(() => {
             store.getActions().should.eql(expectedActions)
-          })
+            done()
+          }).catch((err) => done(err))
```

Comment:
Just a little test cleanup here-- without this `.catch`, errors will be logged in the output but won't explicitly fail the test.